pixi/mocks/pageExperienceCheck/reportDataErrors.json

```diff
@@ -1,44 +1,6 @@
 {
 	"pageExperience": {
-		"fieldData": {
-			"lcp": {
-				"unit": {
-					"name": "s",
-					"conversion": 1000,
-					"digits": 1
-				},
-				"data": {
-					"numericValue": 3100,
-					"category": "AVERAGE",
-					"proportion": 0.6472574979735206
-				}
-			},
-			"fid": {
-				"unit": {
-					"name": "ms",
-					"conversion": 1,
-					"digits": 0
-				},
-				"data": {
-					"numericValue": 289,
-					"category": "AVERAGE",
-					"proportion": 0.09960294951786727
-				}
-			},
-			"cls": {
-				"unit": {
-					"name": "",
-					"conversion": 100,
-					"digits": 2
-				},
-				"data": {
-					"numericValue": 0,
-					"category": "FAST",
-					"proportion": 0.932653831634579
-				}
-			},
-			"isAllFast": false
-		},
+		"fieldData": null,
```

Comment:
We should introduce a new test series here to check the case when origin_fallback is set and change the existing error case to not have origin_fallback set.
Otherwise we never check the "not fast" cases in the field data together with "isAllFast: false"

Reply:
Good point, reset the changes to the error test case.